gh-pages-stub/_includes/nav.html

```diff
@@ -17,11 +17,11 @@
           {% assign postfix = ""  %}
         {% endif %}
           {% if val.url == navurl %}
-            <li class="menu-selected menu-opened"><a href="{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
+            <li class="menu-selected menu-opened"><a href="{{ site.baseurl }}{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
 		  {% elsif navurl contains val.url %}
-			<li class="menu-opened"><a href="{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
+			<li class="menu-opened"><a href="{{ site.baseurl }}{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
           {% else %}
-            <li><a href="{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
+            <li><a href="{{ site.baseurl }}{{ val.url }}">{{ postfix }}{{ val.text }}</a></li>
```

Comment:
can this template do something like {{global.variable1 + global.variable2 }} ? or only {{global.variable1}}{{global.variable2}} ?